### PR TITLE
Add a search page

### PR DESCRIPTION
**Pull request by gregory-latinier**
There is a `LIKE % search %` that is set up on the fields : username, email, phone and IP

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -212,4 +213,68 @@ router.post('/reject', authenticate(), routeMiddleware(async (req) => {
...
+    items: elements
+  });
+});
+
+router.post('/search', authenticate(), routeMiddleware(async (req) => {
+  const page = parseInt(req.body.page) || 1;
+  const search = req.body.search;
+  const Op = Sequelize.Op;
+  const where = {
+    [Op.or]: [
```
Comment (bonustrack) [inline routes/index.js]: Can we have the params within a single line like this:
```
{ email: { [Op.like]: `%${search}%` } },
{ username: { [Op.like]: `%${search}%` } },
...
```

File: routes/index.js [hunk 2]
```diff
@@ -212,4 +213,68 @@ router.post('/reject', authenticate(), routeMiddleware(async (req) => {
...
+          [Op.like]: `%${search}%`
+        },
+      },
+      {
+        phone_number: {
+          [Op.like]: `%${search}%`
+        },
+      },
+      {
+        ip: {
```
Comment (bonustrack) [inline routes/index.js]: Can you also add the field `fingerprint` with like search?

**Comment by bonustrack** [comment]
@gregory-latinier We need the status "All" aswell.

**Comment by jnordberg** [comment]
Needs a date picker widget though but let's do that in a separate pr

**Comment by jnordberg** [comment]
Closes #36

**Pull request merged**